validator/README.md

```diff
@@ -0,0 +1,112 @@
+# Validator
+
+The validator performs two main tasks:
```

Comment:
Thanks for writing such through documenation. How are you planning on integrating this with the rest of the system -- will this be added as an extra automation step?

Reply:
Yes. The intended pipeline is:
1. Run indicator
2. Run validator <- this package
3. Run archiver <- [completed but not yet integrated](https://github.com/cmu-delphi/covidcast-indicators/pull/295)